evap/evaluation/templates/base.html

```diff
@@ -81,6 +95,11 @@
             };
             activateTooltips();
 
+
+            for(const offCanvasElement of document.querySelectorAll('.offcanvas')) {
+                new bootstrap.Offcanvas(offcanvasEl);
+            }
```

Comment:
same discussion as with the Collapsible -- just from reading the [bootstrap documentation](https://getbootstrap.com/docs/5.0/components/offcanvas/), this shouldn't be necessary, bootstrap should already do its thing with anything that has `class="offcanvas"`.

Reply:
fixed